lib/Decompressor.js

```diff
@@ -63,12 +106,7 @@ export class Decompressor extends Transformer {
    */
   async decompress({compressedBytes, diagnose} = {}) {
     this.contextMap = new Map();
-    this.termToId = new Map(KEYWORDS);
-    this.nextTermId = FIRST_CUSTOM_TERM_ID;
-    this.idToTerm = new Map();
-    for(const [term, id] of this.termToId) {
-      this.idToTerm.set(id, term);
-    }
+    this.nextTermId = 100;
```

Comment:
I'm not sure this behavior is the same as before. It seems that we're not resetting things? If we don't ever expose the Decompressor as a public API ... then I guess we can change this behavior and make them "one offs".

However, if we are already exposing it externally (as a symbol someone can import) then we'll need to change this back to get similar behavior to before. In that case, I imagine we should add a test that calls `decompress()` on the same instance, using two different inputs in different orders to see that the results aren't the same for each of those inputs. We'll want to fix this up to call an internal `reset()` function or something that minimally resets the `idToTerm` and `termToId` tables.

And we should use the constant here instead of `100`. Of course, if we're doing a "one off" now (and we don't have to do the above fix), we should also just set this once in the `Transformer` constructor and not both setting it here again.

```suggestion
    this.nextTermId = FIRST_CUSTOM_TERM_ID;
```

Reply:
_reset() added. TODO add test.